app/templates/notification.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each model as |post|}}
+  <NotificationCard @title = {{post.title}} @description = {{post.description}}/>
```

Comment:
Please remove the space after and before "=". Also make sure to leave space before ending of the tag " />".